Fixing (most) quest, archaeology & diary issues

SnowyToo opened the pull request:
~Diary task fixes following shortly~
E: Diary fixes implemented. Underworld diaries currently not all doable by vanilla (Kili's tasks don't show up for vanilla's + Anachronia content necessary)

SnowyToo committed:
Quest changes 11/03/24

DoD: Defender of Varrock & WGS

SnowyToo committed:
All quests + zorgoth rings + manifest ectoplasm

Also changed all rituals to require a ritual site instead of just Um

SnowyToo committed:
missing commas

haha happens to the best of us right
right

SnowyToo committed:
Underworld Achievements added

SnowyToo committed:
Sanctum of Rebirth + misc fixes

SnowyToo committed:
Harrowed Lands fix, PR fixes, typos

SnowyToo committed:
Merge branch 'gh-pages' into learner-gnome

SnowyToo committed:
Most of the quests & archaeology bugs fixed

SnowyToo committed:
Fixed outputs in last commit

ThaFrenchFry commented on rs3-chunkpicker-chunkinfo-export.json:
```diff
@@ -4405,14 +4445,15 @@
 				"Priority": 7
 			},
-			"Clear the ~|Barricade (Archaeology)|~ in the Warforge": {
+			"Clear the ~|barricade (Archaeology)|~ in the Warforge": {
```
Change to `barricade (Warforge Dig Site)` in 3 spots

ThaFrenchFry commented on rs3-chunkpicker-chunkinfo-export.json:
```diff
@@ -88423,7 +88511,6 @@
 					"~|Desert Treasure|~ Complete the quest": "Quest",
 					"~|Horror from the Deep|~ Complete the quest": "Quest"
 				},
-				"QuestPointsNeeded": 176,
```
Revert this

ThaFrenchFry commented on rs3-chunkpicker-chunkinfo-export.json:
```diff
@@ -79943,6 +80029,7 @@
 				"Kudos": 5,
 				"BaseQuest": "In Aid of the Myreque",
 				"Chunks": ["Hallows Myreque Hideout"],
+				"Reward": ["Nail beast nails"],
```
You can do better :)

ThaFrenchFry commented on rs3-chunkpicker-chunkinfo-export.json:
```diff
@@ -92296,7 +92383,6 @@
 				"Skills": {
 					"Prayer": 50
 				},
-				"KudosNeeded": 125,
```
You can also do better :)

ThaFrenchFry commented on rs3-chunkpicker-chunkinfo-export.json:
```diff
@@ -114338,273 +114427,273 @@
+				"Chunks": ["13108", "13364", "13365"]
 			},
-			"Complete the archaeolgy collection 'Anarchic Abstraction'":{
-				"Items": ["'Disorder' painting", "'The Lake of Fire' painting", "'Pandemonium' tapestry"],
```
Undo

ThaFrenchFry commented on rs3-chunkpicker-chunkinfo-export.json:
```diff
@@ -114338,273 +114427,273 @@
 				"NPCs": ["Sir Atcha"]
 			},
-			"Complete the archaeolgy collection 'Zarosian I'":{
-				"Items": ["Venator dagger", "Venator light crossbow", "Primis Elementis standard", "Legionary gladius", "Legionary square shield", "Zaros effigy", "Zarosian training dummy", "Legatus Maximus figurine", "'Solem in Umbra' painting"],
```
undo

ThaFrenchFry commented on rs3-chunkpicker-chunkinfo-export.json:
```diff
@@ -158280,8 +158381,8 @@
 				"Revenant bane off hand longsword"
 			],
 			"Barbarian bait[+]": [
-				"feathers+",
```
Add `Feathers[+]`

SnowyToo replied:
Can't nest itemsPlus. Have added colored feathers to this list.

SnowyToo committed:
Powerful ink fix + Master cape fixes

SnowyToo committed:
Temple Trekking, Kudos, typos

SnowyToo commented:
Closed unintentionally

SnowyToo committed:
Fixed automatons & dragonkin anvil; added pickaxe of E&S

SnowyToo committed:
Fixing (most) diary tasks

Fungal Fun Ghoul requires Anachronia content. Not touching that now.

Raptor still not showing up on slayer tab.

SnowyToo commented on rs3-chunkpicker-chunkinfo-export.json:
```diff
@@ -308776,7 +309045,7 @@
 			}
 		},
-		"Raptor": {
+		"The Raptor#Fort Forinthry": {
```
Raptor does not show up on Slayer tab in the picker currently. Could this be related to the NPC being task-locked behind Unwelcome Guests, or am I just missing something obvious?

SnowyToo committed:
Fixed Cres automata

but like for real this time

SnowyToo committed:
Elite Desert diary fix

SnowyToo committed:
ED fixes, Primal stone spirits added, typos

SnowyToo committed:
Energy potion () lamayo
